protocol/x/bridge/keeper/acknowledge_bridges.go

```diff
@@ -20,6 +20,14 @@ func (k Keeper) GetAcknowledgeBridges(
 	ctx sdk.Context,
 	blockTimestamp time.Time,
 ) (msg *types.MsgAcknowledgeBridges) {
+	// Do not propose bridge events if bridging is disabled.
+	safetyParams := k.GetSafetyParams(ctx)
```

Comment:
meganit, totally optional:
`if k.GetSafetyParams(ctx).IsDisabled {`

(I think you follow this elsewhere where you can.)